.idea/.gitignore

```diff
@@ -0,0 +1,11 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+
+# for jetbrains users
+.idea/
```

Comment:
is the nested `.idea` needed here?

Reply:
Sorry, I have no idea how that got in there. It should be gone now.